shared_model/validators/CMakeLists.txt

```diff
@@ -12,11 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-add_library(shared_model_stateless_validation
-        default_validator.cpp
+
+add_library(shared_model_field_validator
+        field_validator.cpp)
+
+target_link_libraries(shared_model_field_validator
+        schema
+        model_interfaces
         )
 
+add_library(shared_model_stateless_validation
+        default_validator.cpp)
```

Comment:
Pls, add an end line after the source. Such as:
```
default_validator.cpp
)
```
This approach useful for future changes in file - line will not appear until change a `default_validator.cpp` directly.